packages/forms/src/Components/Repeater.php

```diff
@@ -111,6 +111,26 @@ function (Repeater $component, string $statePath, string $uuidToMoveUp): void {
 
                     $items = array_move_before($component->getState(), $uuidToMoveUp);
 
+                    $livewire = $component->getLivewire();
+                    data_set($livewire, $statePath, $items);
+                },
+            ],
+            'repeater::moveItems' => [
```

Comment:
Since we add this new event listener for moving items, can we remove the old up/down button listeners for Repeater & Builder?

Reply:
Maybe I am late in the conversation, but would it be possible to have the Drap&Drop placement driven by config. I don't know the complexity in implementing that

Reply:
The up/down listeners are still used by the arrow-up/arrow-down key events on the drag handle. Without those you wouldn't be able to move items via the keyboard.